static/js/interactive-map/NewMapConfig.js

```diff
@@ -0,0 +1,215 @@
+import { Coordinate } from './Geo/Coordinate.js';
+import { PinImages } from './PinImages.js';
+import { ClusterPinImages } from './ClusterPinImages.js';
+import { getLanguageForProvider } from './Util/helpers.js';
+import { defaultCenterCoordinate } from './constants.js';
+
+/**
+ * The configuration for the NewMap component.
+ */
+export default class NewMapConfig {
```

Comment:
I think this change will be very helpful. We may want to change this class name as part of the "Change names of components" item. We want the HHs to understand that the config they specify in the JSON will be passed to this class, then this class might do some additional work to it.

Reply:
Agreed! Added a reminder to SLAP-1105